grails-app/controllers/org/pih/warehouse/core/ErrorsController.groovy

```diff
@@ -97,7 +98,7 @@ class ErrorsController {
             response.status = 400
             BeanPropertyBindingResult errors = request?.exception?.cause?.errors
             def errorMessages = errors.allErrors.collect {
-                return messageService.getMessage(it.codes[0], it.arguments, (it.defaultMessage ?: it.codes[0]), null)
+                return messageService.getMessage(it.codes[0], it.arguments, (it.defaultMessage ?: it.codes[0]), localizationService.getCurrentLocale())
```

Comment:
@jmiranda Passing `null` here was causing the NullpointerException in the spring's getMessage(). This is "temporary" fix. I add a topic regarding this to the Monday's TH.